regressionfiles.txt

```diff
@@ -150,7 +150,6 @@ GAMESS/GAMESS-US2014/water_cis_dets.out
 GAMESS/GAMESS-US2014/water_cis_saps.out
 GAMESS/GAMESS-US2014/water_mp2.out
 GAMESS/GAMESS-US2018/exam45.log
-GAMESS/GAMESS-US2018/exam46.log
```

Comment:
Was this a mistake?

Reply:
sorry yeah, don't know how I managed that...